### DISCUSSION TITLE
Controller input support?

### DISCUSSION THREAD

**Discussion by TheMathGeek314** [Elringus/UnityRawInput#25] [Q&A] [answered]
Will this project ever include external controller inputs? I'm working on a project that really needs to run controller input through a background unity program and haven't found anything capable of doing this.

**Comment by elringus** [comment, marked as answer]
That's unlikely, as I'm not using this project myself; but if someone will make a decent PR, I can take a look. Mouse support was added this way (#18).

**Comment by BuckieBlu** [comment]
Yes please can someone add support for like, Logitech G29 Steering Wheels? I'd die for this >A<
